openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```
@@ -309,6 +309,7 @@
     "detail-plural": "Details",
     "ingestion": "Ingestion",
     "add-workflow-ingestion": "Add {{workflow}} Ingestion",
+    "ingestion-workflow-action": "{{action}} {{workflow}} Ingestion",
```

```suggestion
    "edit-workflow-action": "edit {{workflow}} Ingestion",
```
as Add one is just present above we can have it as Edit one